packages/nvidia_nat_core/tests/nat/runtime/test_session_manager.py

```
@@ -498,6 +516,128 @@ async def test_cleanup_skips_active_builders(self, mock_registry):
+                            shared_workflow=MockWorkflow())
+
+        async with sm.session() as session:
+            # Before run, parent context should be unset
+            assert ctx_state.workflow_parent_id.get() is None
+            assert ctx_state.workflow_parent_name.get() is None
+
+            values_during_run = []
+
+            async with session.run("hello",
+                                  parent_id="parent-step-123",
+                                  parent_name="Caller Workflow") as runner:
```

Ruff (0.15.0)

[error] Local variable `runner` is assigned to but never used

Remove assignment to unused variable `runner`

(F841)